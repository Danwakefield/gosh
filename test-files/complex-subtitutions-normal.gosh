FULL=SUCCESS
EMPTY=
echo ${FULL}
echo "The next two lines should be empty"
echo ${EMPTY}
echo ${NOTSET}
