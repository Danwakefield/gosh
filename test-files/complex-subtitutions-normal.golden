SUCCESS
The next two lines should be empty


